QSB/TornadoSync/TornadoManager.cs

```diff
@@ -0,0 +1,49 @@
+﻿using QSB.TornadoSync.TransformSync;
+using QSB.TornadoSync.WorldObjects;
+using QSB.Utility;
+using QSB.WorldSync;
+using QuantumUNET;
+
+namespace QSB.TornadoSync
+{
+	public class TornadoManager : WorldObjectManager
+	{
+		protected override void RebuildWorldObjects(OWScene scene)
+		{
+			QSBWorldSync.Init<QSBTornado, TornadoController>();
+
+			if (!QSBCore.IsHost)
+			{
+				return;
+			}
+
+			foreach (var transformSync in QSBWorldSync.GetUnityObjects<OccasionalTransformSync>())
+			{
+				QNetworkServer.Destroy(transformSync.gameObject);
+			}
+
+			var gdBody = Locator._giantsDeep.GetOWRigidbody();
+			// cannon
+			var cannon = Locator._orbitalProbeCannon.GetRequiredComponent<OrbitalProbeLaunchController>();
+			SpawnOccasional(cannon.GetAttachedOWRigidbody(), gdBody);
+			foreach (var proxy in cannon._realDebrisSectorProxies)
+			{
+				SpawnOccasional(proxy.transform.root.GetAttachedOWRigidbody(), gdBody);
```

Comment:
no other way to sync these without network transforms? :/

Reply:
not without hassle, no